lib/is_it_working/status.rb

```diff
@@ -63,5 +67,10 @@ def success?
     def warnings?
       success? && @messages.any?{|m| m.warn?}
     end
+
+    # Returns +true+ if any checks were FAIL.
+    def failures?
+      @messages.any?{|m| m.fail?}
```

Comment:
could this just be `@messages.any?(&:fail?)`?

Reply:
I had cargo-culted the `success?` method up above but will update all 3

Reply:
803d9b3